guides/common/modules/proc_installing-and-configuring-the-command-line-assistant-powered-by-rhel-lightspeed.adoc

```diff
@@ -0,0 +1,51 @@
+[id="installing-and-configuring-the-command-line-assistant-powered-by-rhel-lightspeed"]
+= Installing and configuring the command-line assistant powered by RHEL Lightspeed
+
+You can install the command-line assistant on your {RHEL} hosts and proxy the assistant traffic through your {ProjectServer} or {SmartProxyServer}.
+
+The command-line assistant powered by RHEL Lightspeed is an optional AI tool available within the RHEL command-line interface trained on Red Hat knowledge from Knowledge Centered Service (KCS) articles, RHEL documentation, and other Red{nbsp}Hat resources.
+The command-line assistant provides you with interactive workflows to solve issues, implement new RHEL features, find information, and more.
+
+You have several options how you can install the command-line assistant:
+
+During host registration::
+You can run the installation and configuration commands in a snippet that is executed after the host is registered by using global registration.
```

Comment:
This reads confusingly to me -- as this is "during host registration" and then says "after the host is registered"